app/lib/blocs/like_animation.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/cupertino.dart';
+
+class LikeAnimation {
```

Comment:
I don't think it goes in bloc directory as this animation doesn't uses bloc styles. Rather have an ```animations``` folder in ```common``` where we can put animations of all widgets.